homeassistant/components/homewizard/config_flow.py

```diff
@@ -28,6 +28,21 @@ def __init__(self) -> None:
         """Initialize the HomeWizard config flow."""
         self.config: dict[str, str | int] = {}
 
+    async def async_step_import(self, import_config: dict) -> FlowResult:
+        """Handle a flow initiated by older `homewizard_energy` component."""
+        _LOGGER.debug("config_flow async_step_import")
+
+        self.hass.components.persistent_notification.async_create(
```

Comment:
Please import a name as normal from in this case the persistent_notification integration instead of accessing `hass.components`. The latter is legacy.

Reply:
I don’t really get what you mean. Import ‘homewizard_energy’ or the ‘persist_notification’?

i copies the persist_notification implementation from the pr that I’ve referenced, so if you want something else please let me know.

Reply:
at the top of the code, `from hass.components.persistent_notification import async_create` then `async_create(` - I need to fix this in the upstream PR

Reply:
Got it, thanks!

Reply:
I did a search in core and found that `from hass.components.persistent_notification import async_create` was not used, but `from homeassistant.components import persistent_notification` was. I think the latter is the best method.